README.md

```diff
@@ -437,13 +440,13 @@ details them.
 
 ## Note on attachments
 
-TLDR; If you backup a database that contains attachments you will not be able to restore it.
+TLDR; If you backup a database that has attachments couchrestore cannot restore it.
```

Comment:
```suggestion
TLDR; If you backup a database that has attachments `couchrestore` cannot restore it.
```

Reply:
We have a mix of plain text and monospaced throughout the README, would you like to see `couchbackup` and `couchrestore` everywhere?

Reply:
Yea, would be nice to have monospaced everywhere but up to you

Reply:
Done in 56a61b2 (I also removed  a TXE caveat that is obviously no longer relevant).

Reply:
Thanks and good catch!